en/controllers/request-response.rst

```diff
@@ -573,7 +573,7 @@ ics generated on the fly from a string::
     {
         $icsString = $this->Calendars->generateIcs();
         $response = $this->response;
-        $response->body($icsString);
+        $response = $response->withStringBody($icsString);
```

Comment:
That's good. Even better would be, if we showed how it needs to be done before 3.4.0:

```` php
        $response = $this->response;
        
        // Inject string content into response body (3.4.0+)
        $response = $response->withStringBody($icsString);
        // Inject string content into response body (before 3.4.0)
        $response->body($icsString);

        $response = $response->withType('ics');
````